api-reference/docstubs/api/activitystatistics-list.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "List activityStatisticses"
+description: "List properties and relationships of the activityStatistics objects."
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# List activityStatisticses
+
+Namespace: microsoft.graph
+
+List properties and relationships of the [activityStatistics](../resources/activitystatistics.md) objects.
```

Comment:
Can we simplify the boilerplate text here to "Get a list of activityStatistics objects."?

Reply:
+1 here.  In a standard GET (get or list) you won't get the relationships by default (you'd need to add a $expand). So definitely agree.

Reply:
I agree with removing "relationships". 
However, IMO, "get a list of x objects" is not providing any extra information from what the topic title says; it's almost like forming a full sentence using the words in the topic title.
I would suggest a boilerplate sentence like "List the properties of activityStatistics objects in a collection."

Reply:
How about:  Get a list of {object link} objects and their properties.

Reply:
Can do.